.env.development

```diff
@@ -0,0 +1,7 @@
+NEXT_PUBLIC_ENVIRONMENT=local
+NEXT_PUBLIC_GA_ID=G-LGFTVFGLKX
+
+# Get this from this message https://discord.com/channels/658523382442360866/903771050385502239/904216952141152328
+DISCORD_WEBHOOK=$SECRET_DISCORD_WEBHOOK
+GITHUB_PAT=$SECRET_GITHUB_PAT
+API_KEY=$SECRET_API_KEY
```

Comment:
I'm guessing the slight change you mentioned would be to set `SECRET_DISCORD_WEBHOOK`, etc in the `.env.local` file?

Is it worth adding a comment here to say that peeps should do that?